rasa/core/actions/action.py

```diff
@@ -830,7 +827,9 @@ def name(self) -> Text:
 
 class ActionExecutionRejection(RasaException):
     """Raising this exception will allow other policies
-    to predict a different action"""
+    to predict a different action
+    .
```

Comment:
I guess autoformat a little failed here, `.` should be on the same line as a sentence.

Reply:
thanks!